.github/workflows/pr-tests-command.yaml

```diff
@@ -24,6 +24,6 @@ jobs:
 
             [1]: ${{ steps.vars.outputs.run-url }}
   test:
-    uses: pulumi/pulumi-str/.github/workflows/stage-test.yml@main
+    uses: ./.github/workflows/stage-test.yml
```

Comment:
It was probably a full path so it couldn't be overridden by the PR itself, so "pulumi/pulumi-std/.github/workflows/stage-test.yml@main"

Reply:
Done :heavy_check_mark: